sympy/solvers/diophantine/diophantine.py

```diff
@@ -536,15 +554,17 @@ def solve(self, parameters=None, limit=None) -> DiophantineSolutionSet:
         elif discr == 0:
 
             if A == 0:
-                s = BinaryQuadratic(self.equation, free_symbols=[y, x]).solve(parameters=[t, u])
+                s = BinaryQuadratic(self.equation,
+                                    free_symbols=[y,
+                                                  x]).solve(parameters=[t, u])
```

Comment:
This seems like an odd choice of line breaks

Reply:
it's left -aligning elements at the opening bracket position if there is more than 1. It's kind of nice because you can scan vertically to see what the given brackets args are -- breadth first.

Reply:
Tbh I don't understand the formatting here :S It looks like this also with the default yapf profile... But with `split_before_expression_after_opening_paren = True`, you get:

```python
                s = BinaryQuadratic(
                    self.equation, free_symbols=[y, x]
                ).solve(parameters=[t, u])
```

See d864ba227c

Reply:
I guess this is another situation where an intermediate variable with a good name would be better